src/style.css

```diff
@@ -0,0 +1,135 @@
+/* #region reset */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body,
+div,
+h2,
+p,
+img {
+  margin: 0;
+  padding: 0;
+  font-family: roboto, arial, sans-serif;
+}
+/* #endregion */
+
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 0;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --dark-color: #060b35;
+  --light-color: #616070;
+  --blue-accent: #00acdc;
+}
+
+.card__img {
+  display: flex;
+  justify-content: center;
+  height: 134px;
+  width: 160px;
+  margin-left: 3px;
+}
+
+.card__name {
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--dark-color);
+  padding-top: 40px;
+  line-height: 18px;
+}
+
+.card__code {
+  font-weight: 400px;
+  font-size: 10px;
+  color: var(--light-color);
+  line-height: 14px;
+  padding-top: 4px;
+}
+/* #endregion */
+
+/* #region review */
```

Comment:
Remove those comments, they are not needed as class names speak for themself good enough already